catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/provides.xml

```diff
@@ -33,7 +33,7 @@ Services (OSGi) that catalog-ui-search PROVIDES to DDF
 
     <!-- Exposing endpoint util as a service to allow for system templates to be managed through
     the means of a Karaf command (reference injection) -->
-    <service ref="endpointUtil" interface="org.codice.ddf.catalog.ui.util.EndpointUtil"/>
+    <service ref="endpointUtil" interface="org.codice.ddf.catalog.ui.api.EndpointUtility"/>
```

Comment:
✏️ If we split the class up, then this can remain unchanged, and we can publish a separate service for the code that needs to be invoked elsewhere.